src/Catty/Setup/CatrobatSetup+Bricks.swift

```diff
@@ -88,6 +88,7 @@
             ThinkForBubbleBrick(),
             ThinkBubbleBrick(),
             AskBrick(),
+            SetLookByIndexBrick(),
```

Comment:
Please order this Brick below the "Switch to look" Brick.

Reply:
could not find "Switch to look" ordered it under "PreviousLookBrick"

Reply:
This is the second Brick in this category, internally referred to as "SetLookBrick".